framework_tvb/tvb/adapters/analyzers/node_complex_coherence_adapter.py

```diff
@@ -81,8 +156,10 @@ def get_filters():
     def get_input_name():
         return "time_series"
 
-    def get_traited_datatype(self):
-        return NodeComplexCoherence()
+
+def result_shape(input_shape, max_freq, epoch_length, segment_length, segment_shift, sample_period, zeropad,
+                 average_segments):
```

Comment:
Why is this function here?

Reply:
I don't know, must have been a mistake...